Package.swift

```diff
@@ -11,5 +11,6 @@ let package = Package(
   targets: [
     .target(name: "Spectre", dependencies: []),
     .testTarget(name: "SpectreTests", dependencies: ["Spectre"]),
-  ]
+  ],
+  swiftLanguageVersions: [.v4, .v4_2]
```

Comment:
v4 doesn't include 4.2?

Reply:
Would this break support on 4.0 because it would have no idea about the `.4_2` enumeration?

Reply:
> v4 doesn't include 4.2?

If you don't add 4.2 explicitly SwiftPM will assume 4 only

> Would this break support on 4.0 because it would have no idea about the .4_2 enumeration?

It depends on what you mean by 4.0 here. It will require the Swift 4.2 (or newer) compiler, but it can still compile with Swift 4.0 syntax. Alternatively I can create a new `Package@swift-4.2.swift` file, and not touch this one, to support both simultaneously going forward.

Reply:
Updated with the new file instead